docs/en/stack/ml/get-started/ml-gs-results.asciidoc

```diff
@@ -50,7 +50,8 @@ display only a single time series.
 The blue line in the chart represents the actual data values. The shaded blue
 area represents the bounds for the expected values. The area between the upper
```

Comment:
Since we're talking about the shaded area in this sentence, I think it makes sense to add the new info here too:
```suggestion
area represents the bounds for the expected values, which is also known as the _confidence interval_. The area between the upper
```